sdk/modelsrepository/Azure.Iot.ModelsRepository/README.md

```diff
@@ -0,0 +1,89 @@
+# Azure IoT Models Repository client library for .NET
+
+This library provides functionality for interacting with the Azure IoT Models Repository. It also aims to provide a consistent experience working with digital twin model repositories following Azure IoT conventions.
+
+[Source code][source] | [Package (nuget)][package]
+
+## Getting started
+
+The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure downloads][microsoft_sdk_download] page, and it ships with support for building deployment packages, integrating with tooling, rich command line tooling, and more.
+
+For the best development experience, developers should use the official Microsoft NuGet packages for libraries. NuGet packages are regularly updated with new functionality and hotfixes.
+
+### Install the package
+
+Install the Azure IoT Models Repository client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+Install-Package Azure.Iot.ModelsRepository
+```
+
+View the package details at [nuget.org][modelsrepository_nuget].
+
+### Prerequisites
+
+- A models repository following [Azure IoT conventions][modelsrepository_conventions]
+  - The models repository can be located on the local filesystem or hosted on a webserver.
+  - Azure IoT hosts the global [Azure IoT Models Repository](modelsrepository_iot_endpoint) which the client will point to by default if no Uri is provided.
+
+### Authenticate the Client
+
+Currently no authentication mechanisms are supported in the client. The global endpoint is not tied to an Azure subscription and does not support auth. All models published are meant for anonymous public consumption.
+
+## Key concepts
+
+The Azure IoT Models Repository enables builders to manage and share digital twin models. The models are JSON-LD documents defined using the Digital Twins Definition Language (DTDL).
+
+The repository defines a pattern to store DTDL interfaces in a directory structure based on the Digital Twin Model Identifier (DTMI). You can locate an interface in the repository by converting the DTMI to a relative path. For example, the DTMI "dtmi:com:example:Thermostat;1" translates to /dtmi/com/example/thermostat-1.json.
```

Comment:
the dtmi id and the translated file path should be in code format

Reply:
done